src/MouseBackend.js

```diff
@@ -195,6 +210,16 @@ export default class MouseBackend {
     this.actions.endDrag()
   }
 
+  handleWindowClick(e) {
+    if (this.preventClick) e.stopPropagation()
```

Comment:
Sorry for the delay in responding! The problem I was solving is that a drag would also fire a click event at the end, when the left mouse was released. My app also needed to handle click events, so it was a big problem that every drag would also fire a click at the end.

I'm not sure why this would cause problems in other apps, but the drag area or behavior might be a bit different, depending on how you use the library.